### PR TITLE
Change query log config format from JSON to TABLE

### PR DESCRIPTION
**Pull request by bhupendray-yb**
## Summary
- Change DB query log config format from JSON to TABLE.
- Show `Integration Name` instead of `Integration ID` in output.
- Fixed unit tests
- Added a new test for `update` log config subcommand flow, when changing the integration name/id for an existing log export config.
- Added support for `json/pretty` output format.

### Old Output:
```shell
State      Integration ID                         Log Config
ENABLING   9e740000-331b-4dec-89b0-4e59b81e9019   {"debug_print_plan":false,"log_connections":false,"log_disconnections":false,"log_duration":false,"log_error_verbosity":"DEFAULT","log_line_prefix":"\%m :\%r :\%u @ \%d :\[\%p\] :","log_min_duration_statement":-1,"log_min_error_statement":"ERROR","log_statement":"NONE"}
```
### New Output:
```shell
# in table format
State     Integration Name
ACTIVE    bhupendray-db-logging

Log Config Key               Log Config Value
debug-print-plan             false
log-min-duration-statement   42
log-connections              false
log-disconnections           false
log-duration                 true
log-error-verbosity          DEFAULT
log-statement                ALL
log-min-error-statement      ERROR
log-line-prefix              %m :%r :%u @ %d %a :[%p] :

# In JSON format
$ make build && ./ybm cluster db-query-logging describe --cluster-name bhupendray-cluster -ojson
{"info":{"cluster_id":"8f0adf54-cc93-4298-b942-36bb669c3fe7","id":"f7933fd1-5e3c-4c92-a4a2-73853737d3dd","metadata":{"created_on":"2024-10-07T10:28:55.148Z","updated_on":"2024-10-22T11:16:49.373Z"},"state":"ACTIVE"},"spec":{"export_config":{"debug_print_plan":false,"log_connections":false,"log_disconnections":false,"log_duration":false,"log_error_verbosity":"DEFAULT","log_line_prefix":"%m :%r :%u @ %d :[%p] :","log_min_duration_statement":-1,"log_min_error_statement":"ERROR","log_statement":"NONE"},"exporter_id":"7eb1b884-600e-4a91-a92c-9ee79b754e11"}}
```

### CODE REVIEW

File: cmd/test/fixtures/db-query-log-exporter-custom.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
           "log_min_error_statement": "ERROR",
           "log_line_prefix": "%m :%r :%u @ %d :[%p] : %a :"
         },
-        "exporter_id": "9e740000-331b-4dec-89b0-4e59b81e9019"
+        "exporter_id": "129f7c97-81ae-47c7-8f9e-40ab4390093f"
```
Comment (bhupendray-yb) [inline cmd/test/fixtures/db-query-log-exporter-custom.json:15]: **Note for reviewer**: I am re-using some response.json files from `integrations_test.go`. So I have changed the exporter id in log exporter json's to same as in `list-telemetry-providers.json` to do some assertions on exporter id.

**Review by bansal01yash — APPROVED** [review]

**Review by Siddarth-Baldwa — APPROVED** [review]
Minor nit. Rest lgtm

File: cmd/cluster/log-exporter/query_log_exporter.go [hunk 2]
```diff
@@ -220,15 +211,21 @@ var updateLogExporterConfigCmd = &cobra.Command{
...
-			integrationName, _ := cmd.Flags().GetString("integration-name")
+			integrationName, _ = cmd.Flags().GetString("integration-name")
 			integrationId, err = authApi.GetIntegrationIdFromName(integrationName)
 			if err != nil {
 				logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
 			}
 		} else {
 			integrationId = logExporterData.Spec.ExporterId
+			integrationName, err = authApi.GetIntegrationNameFromId(integrationId)
+			fmt.Println("making get call")
```
[the remark this answers is not in the document]
Comment (bhupendray-yb) [reply cmd/cluster/log-exporter/query_log_exporter.go]: fixed.

**Review by posriniv — APPROVED** [review]
Looks good to me!

**Pull request merged**